drivers/spi/spi_cc13xx_cc26xx.c

```diff
@@ -119,6 +119,18 @@ static int spi_cc13xx_cc26xx_configure(struct device *dev,
 		sys_set_bit(cfg->base + SSI_O_CR1, 0);
 	}
 
+#if (CONFIG_SPI_CC13XX_CC26XX_LOOPBACK_MODE_0)
+	if (get_dev_config(dev)->base ==
+		DT_TI_CC13XX_CC26XX_SPI_40000000_BASE_ADDRESS) {
+		HWREG(cfg->base + SSI_O_CR1) |= SSI_CR1_LBM;
+	}
+#elif (CONFIG_SPI_CC13XX_CC26XX_LOOPBACK_MODE_1)
+	if (get_dev_config(dev)->base ==
+		DT_TI_CC13XX_CC26XX_SPI_40008000_BASE_ADDRESS) {
+		HWREG(cfg->base + SSI_O_CR1) |= SSI_CR1_LBM;
+	}
+#endif
+
```

Comment:
So I meant that piece of code. 

Looks like there is no difference between lines 125 and 130 thus why the Kconfig options seem useless to me. 
There is no need to do such base address comparison.

Also the line HWREG(cfg->base + SSI_O_CR1) |= SSI_CR1_LBM; should go after line 119. Loopback is only enabled there.

Reply:
Thanks @tbursztyka. Now I see what you meant. You are correct, line 119 is doing the same thing.

It looks like the issue isn't so much that there is no way to enable loopback mode, but rather the loopback test is (surprisingly) not enabling it via SPI_MODE_LOOP. I'll make that change and resubmit.